php7.2/apache/hooks/post_push

```diff
@@ -19,7 +19,7 @@ set -e
 # X
 
 
-for tag in 5.9.0-php7.2-apache 5.9-php7.2-apache 5-php7.2-apache; do
+for tag in 5.9.0-php7.2-apache; do
```

Comment:
memo: muss ich mal in der update.sh checken, warum der die anderen tags da raus ballert.